### PR TITLE
[F2F-15] - Express server and AWS infra setup

### PR DESCRIPTION
**Pull request by mcgarj**
## Proposed changes
Creates the express server with initial view as part of [F2F-15](https://govukverify.atlassian.net/browse/F2F-15) and also the AWS infra setup as part of [F2F-17](https://govukverify.atlassian.net/browse/F2F-17)

[F2F-15]: https://govukverify.atlassian.net/browse/F2F-15?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[F2F-17]: https://govukverify.atlassian.net/browse/F2F-17?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @alphagov/digital-identity-ipv
```
[the remark this answers is not in the document]
Comment (mcgarj) [reply CODEOWNERS:1]: No problem, would that still need the `* @alphagov/` before `di-ipv-kiwi-front-codeowners`, or is it just `di-ipv-kiwi-front-codeowners`

[the remark this answers is not in the document]
Comment (richa-misra) [reply CODEOWNERS:1]: just kiwi please

File: src/lib/config.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    BASE_URL: process.env.EXTERNAL_WEBSITE_HOST || "http://localhost:8000",
+    PATHS: {
+      F2F: "/",
+    },
+    ANALYTICS: {
+      ID: process.env.GTM_ID,
+      DOMAIN: process.env.ANALYTICS_DOMAIN || "localhost",
+    },
+  },
+  PORT: process.env.PORT || 5020,
```
[the remark this answers is not in the document]
Comment (richa-misra) [reply src/lib/config.js:22]: yeh sounds good....would need updating in package.json too

[the remark this answers is not in the document]
Comment (mcgarj) [reply src/lib/config.js]: All done and updated!

File: CODEOWNERS (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcgarj) [reply CODEOWNERS]: Thank you, I've pushed that change now!

**Review by richa-misra — APPROVED** [review]

**Pull request merged**